class-gravity-flow.php

```diff
@@ -5957,6 +5959,32 @@ public function filter_after_update_entry( $form, $entry_id ) {
 			}
 		}
 
+		/**
+		 * Add inbox notification count to Workflow Menu.
+		 *
+		 * @since 2.5.12
+		 * 
+		 * @param array $menu The current WP Dashboard Menu.
+		 */		
+		public function show_notification_count( $menu ) {
```

Comment:
Would this be better?
```suggestion
		public function show_inbox_count( $menu ) {
```